hera_cal/tests/test_io.py

```diff
@@ -480,6 +480,20 @@ def test_iterate_over_bls(self):
         with pytest.raises(NotImplementedError):
             next(hd.iterate_over_bls())
 
+        hd = HERAData(self.uvh5_1)
```

Comment:
This isn't really testing the redundancy of the chunks...

Also, I'd suggest testing Nbls < len(red).

Reply:
(if it's not too hard, your call. I see that a lot of this is tested in `test_chunck_baselines_by_redundant_group`)

Reply:
I can check redundancy here without much difficulty.

Reply:
But it's only easy when `Nbls=1` so in all cases `Nbls < len(red)`.